com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Decal.md

```diff
@@ -62,9 +62,26 @@ To view these properties, click the **Cog** in the top right of the Master Node.
 
 ### Material Properties
 
-These properties are in the **Exposed Properties** section of the Inspector, alongside the properties that you exposed in the Shader Graph's Blackboard. If you set **Override ShaderGUI** to `true`, the Material Properties section does not appear, and instead, the ShaderGUI you specified appears.
+When creating a Decal Material from the Decal MAster node, following properties are available, alongside the properties that you exposed in the Shader Graph's Blackboard.
```

Comment:
When creating a Decal Material from the Decal master node, the following properties are available alongside the properties that you exposed in the Shader Graph's Blackboard.